components/match2/wikis/rainbowsix/match_summary.lua

```diff
@@ -227,15 +276,17 @@ function MapVeto:createHeader()
 	return self
 end
 
+---@param firstVeto number?
+---@return R6MapVeto
```

Comment:
```suggestion
---@return self
```
 I think it more correct? Applies for other places too

Reply:
VSCode/LuaExtension proposes the classname instead of self
if we want self there i can adjust it